pkg/test/resourcefixture/testdata/basic/container/v1beta1/maqiuyu/create.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Google LLC
```

Comment:
It seems you have added four test YAMLs including this one by mistake?

Reply:
Yep, that was by mistake. Removed now!